benchmark/sampler/neighbor.py

```diff
@@ -0,0 +1,62 @@
+import argparse
+import time
+
+import torch
+# uncomment below to enable torch.ops.torch_sparse.neighbor_sample
+# import torch_sparse
```

Comment:
Can we gate this with a `try/catch`, and avoid the `torch_sparse` benchmark in case it was not successfully imported?

Reply:
+1, something like
```python
try:
    import torch_sparse
    baseline_neighbor_sample = torch.ops.torch_sparse.neighbor_sample
except ImportError:
    baseline_neighbor_sample = None
```

Reply:
Thanks for the comments! The problem with this import is that basically torch_sparse is not used in this file and we get an error from pre-commit:
`benchmark/sampler/neighbor.py:7:5: F401 'torch_sparse' imported but unused`
So there are two options: we can comment out this line or create a dummy torch_sparse object. What do You think?

Reply:
Ah, simply do
```python
import torch_sparse  # noqa
```

Reply:
Thanks, done.